src/Analysis/Engine/Test/CompletionTest.cs

```diff
@@ -410,5 +410,23 @@ class Class1(list):
             completion.Should().HaveItem(expectedLabel)
                 .Which.Should().HaveInsertText(expectedInsertText);
         }
+
+        [ServerTestMethod, Priority(0)]
+        public async Task Completion_AddBracketsEnabled_MethodOverride(Server server) {
+            var code = @"
+class A(object):
+    def foo(self):
+        pass
+
+class B(A):
+    def f";
+
+            await server.SendDidChangeConfiguration(new ServerSettings.PythonCompletionOptions { addBrackets = true });
+            var uri = await server.OpenDefaultDocumentAndGetUriAsync(code);
+            var completion = await server.SendCompletion(uri, 6, 9);
+
+            completion.Should().HaveItem("foo")
+                .Which.Should().HaveInsertText("foo(self):\r\n    return super().foo()");
```

Comment:
Just to be clear, these tests run under Python 3, correct? Otherwise, it'd be completed out to `super(A, self).foo()`.

Reply:
May be worth handling both?

Reply:
Done